src/templates/pycontw-2020/contents/en/events/passion-fruit.html

```diff
@@ -0,0 +1,107 @@
+
+{% extends 'contents/_base.html' %}
+
+{% load static %}
+
+{% block title %}Python NPF 百香果{% endblock title %}
+
+{% block meta_description %}Python NPF百香果是今年一個由PyCon Taiwan所籌備的小型線上會議，在疫情肆虐全球的情況下，我們希望藉由這樣的線上活動來拉近彼此間的距離，並利用此本次活動向大家揭露南投.py的神秘面紗！{% endblock meta_description %}
+
+
+{% block content %}
+
+<section class="pill-title alt">
+    <h1>第壹次Python NPF百香果 x 南投.py</h1>
+</section>
+
+<img src="{% static 'pycontw-2020/assets/warm-session/passion-fruit/info.png' %}" width="100%">
```

Comment:
Oh shoot I didn't catch this until I run it on the staging server. It should be `Info.png` instead of `info.png`. Didn't catch this before in mac it is case insensitive. orz. The same goes to the zh version of the file.

Reply:
Please make another fix for this. Thanks! @mattwang44